### PR TITLE
Adding homebrew installation instructions

### PR DESCRIPTION
**Pull request by jenkoian**
Awaiting https://github.com/Homebrew/homebrew-php/pull/2444

### CODE REVIEW

**Comment by webmozart** [comment]
Awesome! :)

**Comment by jenkoian** [comment]
https://github.com/Homebrew/homebrew-php/pull/2444 now merged

File: installation.rst [hunk 1]
```diff
@@ -64,6 +64,17 @@ and enter the following command in a terminal:
...
+OSX Only: Using Homebrew
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are on OSX, you can also install through Homebrew_:
+
+.. code-block:: text
+
+    $ brew install puli
+
+Note: requires `Homebrew PHP to be installed`_
```
Comment (webmozart) [inline installation.rst]: Can you expand to:

> Note: This command requires...

File: installation.rst [hunk 2]
```diff
@@ -64,6 +64,17 @@ and enter the following command in a terminal:
 
     $ composer require --dev puli/cli:^1.0
 
+OSX Only: Using Homebrew
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are on OSX, you can also install through Homebrew_:
```
Comment (webmozart) [inline installation.rst]: > you can also install Puli through

File: installation.rst (hunk 1 above)
Comment (webmozart) [inline installation.rst]: And please add a dot at the end of the sentence.

**Comment by webmozart** [comment]
Nice :) Can you fix my comments? Then this can be merged.

**Comment by jenkoian** [comment]
Done.

**Comment by webmozart** [comment]
Thanks :)

**Pull request merged**